### PR TITLE
Adding e2e test coverage

### PR DESCRIPTION
**Pull request by christian-bromann**
## PR Type

What kind of change does this PR introduce?

```html
[ ] Bugfix
[ ] Feature
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other: adding test coverage
```

## What is the current behavior?

Currently this extension has a set of simple unit tests. Unfortunately there is no assurance that the functionality works across operating systems. Also the tests are run within Travis which has been ceased.

## What is the new behavior?

This patch adds a simple e2e test to ensure that the server starts and a browser is able to connect and receive files. It also moves the test execution to GitHub Action.

## Does this PR introduce a breaking change?

```text
[ ] Yes
[x] No
```

## Other information

I am the maintainer of WebdriverIO and recently started building VSCode extensions, e.g. [Marquee](https://marquee.stateful.com/). I've experienced that there wasn't good tooling for e2e test scenarios that are sometimes needed for complex extensions like yours.

### CODE REVIEW

**Comment by christian-bromann** [comment]
@ritwickdey any comments?

File: test/e2e/example.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+describe('VSCode Live Server Extension', () => {
+    let port: number;
+    let browser: WebdriverIO.Browser;
+
+    before('start browser', async () => {
+        browser = await remote({
+            capabilities: { browserName: 'chrome' }
+        });
+        const workbench = await driver.getWorkbench();
```
Comment (ritwickdey) [inline test/e2e/example.e2e.ts:11]: (Looks good to me but this comment about the API)

 I'm a bit confused here. Does `driver.getWorkbench()` gives vscode instance variable? `driver` is from `webdriverio` package. right?

**Review by ritwickdey — COMMENTED** [review]
Thank you so much for the PR ❤️

File: test/e2e/example.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        /**
+         * wait until all notifications are removed because sometimes
+         * VSCode supresses notifications if others pop up
+         */
+        await workbench.elem.waitUntil(async () => {
+            const notifications = await workbench.getNotifications();
+            for (const n of notifications) {
+                await n.dismiss();
+            }
+            return (await workbench.getNotifications).length === 0;
```
[the remark this answers is not in the document]
Comment (christian-bromann) [reply test/e2e/example.e2e.ts]: good catch, fixed

File: test/e2e/example.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        });
+    });
+
+    it('should click on Go Live', async () => {
+        const workbench = await driver.getWorkbench();
+        await workbench.elem.$('div[id="ritwickdey.LiveServer"]').click();
```
[the remark this answers is not in the document]
Comment (christian-bromann) [reply test/e2e/example.e2e.ts:28]: yes, it is the extension id: `<publisher>.<name>`

**Review by surya9372 — COMMENTED** [review]
Console.log("Hello")

**Pull request merged**